gateway/src/lib.rs

```diff
@@ -17,25 +17,76 @@
 //!
 //! ## Features
 //!
-//! `twilight-gateway` includes only a feature: `simd-json`.
+//! ### Deserialization
+//!
+//! `twilight-gateway` supports [`serde_json`] and [`simd-json`] for
+//! deserializing and serializing events. These features are mutually exclusive.
+//! `serde_json` is enabled by default.
+//!
+//! #### `serde_json`
+//!
+//! [`serde_json`] is the inverse of `simd-json` and will use the `serde_json`
+//! crate to deserialize responses.
+//!
+//! This is enabled by default.
```

Comment:
I reduced this section for `twilight-http` to this one-liner

```suggestion
//! `twilight-gateway` supports [`serde_json`] and [`simd-json`] for
//! deserializing and serializing events.
```